resources/Microsoft.Windows.Setting.Accessibility/Microsoft.Windows.Setting.Accessibility.psm1

```diff
@@ -231,6 +238,56 @@ class MousePointer {
     }
 }
 
+[DSCResource()]
+class AlwaysShowScrollbars {
+    [DscProperty(Key)] [BinarySettingState] $ShowScrollBars = [BinarySettingState]::KeepCurrentValue
+
+    hidden [string] $DynamicScrollbarsProperty = 'DynamicScrollbars'
+
+    [AlwaysShowScrollbars] Get() {
+        $currentState = [AlwaysShowScrollbars]::new()
+        
+		if (-not(DoesRegistryKeyPropertyExist -Path $global:AlwaysShowScrollbarsRegistryPath -Name $this.DynamicScrollbarsProperty)) {
+            $ShowBars = [BinarySettingState]::Disabled                
+        } else {
+			$ShowBars = (Get-ItemProperty -Path $global:AlwaysShowScrollbarsRegistryPath -Name $this.DynamicScrollbarsProperty).DynamicScrollbars
+			$currentState.ShowScrollBars = switch ($ShowBars) {
+				1 { [BinarySettingState]::Disabled }               
+				0 { [BinarySettingState]::Enabled }               
+				default { [BinarySettingState]::KeepCurrentValue }

```

Comment:
Get method should always return the actual state of the resource. My thinking is that if 1 == Disabled, then everything else must be `Enabled`. No need to return KeepCurrentValue here as that does not tell the user what state we are in.

Reply:
Shouldn't a `switch` always have a `default`? Should that be an error state here?